docs/en/05-supported-backends/vacc.md

```diff
@@ -0,0 +1,94 @@
+# VACC Backend
+
+- cmake 3.10.0+
+- gcc/g++ 7.5.0
+- llvm 9.0.1
+- ubuntu 18.04
+
+## PCIE
+
+### 1.package
+
+- dkms (>=1.95)
+- linux-headers
+- dpkg (Ubuntu)
+- rpm  (CentOS)
+- python2
+- python3
+
+Check if there is a vacc card：`lspci -d:0100`
+
+1. Requirements
+
+   ```bash
+   sudo apt-get install dkms dpkg python2 python3
+   ```
+
+2. install driver
+
+   ```bash
+   sudo dpkg -i vastai-pci_xx.xx.xx.xx_xx.deb
+   ```
+
+3. Verify installation
+
+   ```bash
+   dpkg --status vastai-pci-xxx
+
+   #output
+   Package: vastai-pci-dkms
+   Status: install ok installed
+   ……
+   Version: xx.xx.xx.xx
+   Provides: vastai-pci-modules (= xx.xx.xx.xx)
+   Depends: dkms (>= 1.95)
+   Description: vastai-pci driver in DKMS format.
+
+   lsmod | grep vastai_pci
+
+   #output
+   vastai_pci        xxx  x
+   ```
+
+4. Upgrade driver
+
+   ```bash
+   sudo dpkg -i vastai-pci_dkms_xx.xx.xx.xx_xx.deb
+   ```
+
+5. Uninstall driver
+
+   ```bash
+   sudo dpkg -r vastai-pci_dkms_xx.xx.xx.xx_xx
+   ```
+
+### 2.reboot pcie
+
+```bash
+sudo chmod 666 /dev/kchar:0 && sudo echo reboot > /dev/kchar:0
+```
+
+## SDK
+
+### step.1
+
+```bash
+pip install torch==1.8.0 torchvision==0.9.0 torchaudio==0.8.0
+pip install onnx==1.10.0， tqdm==4.64.1
```

Comment:
No ", " between onnx and tqdm
Besides, I am trying to create a conda env and start from scratch.
Adn I found `decorator` and `scipy` are also required.

Reply:
ok, thanks~